cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -181,6 +181,21 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'align', 'justify');
 	});
 
+	it('Apply spacing', function() {
+		cy.get('#tb_editbar_item_linespacing')
+			.click();
+
+		helper.reselectAllText();
+
+		cy.get('.menu >.w2ui-drop-menu>tbody>tr').eq(1)
+			.click();
```

Comment:
Here you should use cy.contains() to get the item you want. It's easier to understand having the string 'Line Spacing: 1.5' in the test code than this long chain of DOM elements.